### ISSUE TITLE
[RFC] Implement block instrumentation

### ISSUE THREAD

**Issue by ChrisBr** [InfluxCommunity/influxdb-rails#108]
It should be possible to wrap code into a block which gets then stored as a single metric

```ruby
InfluxDB::Ralis.instrument "name", tags: { }, values: { } do
  1 + 1
end
```

**Comment by hennevogel** [comment]
Hm, I'm not quite sure how this is different from using influxdb-ruby

https://github.com/influxdata/influxdb-ruby#writing-data

**Comment by hennevogel** [comment]
We also documented this here https://github.com/influxdata/influxdb-rails#custom-client-configuration

**Comment by ChrisBr** [comment]
> Hm, I'm not quite sure how this is different from using influxdb-ruby

Yes true, but I still need to know quite a bit about influxdb-ruby to do so

* `measurement_name` needs to get passed in
*  Need to merge additional values and tags
* The duration value needs to get calculated

So the whole thing ends up rather ugly or I have to write my own helper in each application.

**Comment by hennevogel** [comment]
Will be shipped in 1.0.1
